test

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Local validation workflows and try to build documentation locally
+echo "1/5 Running precommit"
+pre-commit run --all-files
+
+echo "2/5 Validating table schemas"
+pre-commit run --all-files
+echo "Processing file: $file"
+for file in spec/*schema.json; do
+  echo "Validating: $file"
+  frictionless validate $file
+done
+
+echo "3/5 Validating datapackage"
+json-schema-cli validate https://specs.frictionlessdata.io/schemas/data-package.json spec/datapackage.json
```

Comment:
I see jsonschema-cli is now part of requirements.txt, but this script (and the documentation) doesn't do anything to  make sure it's installed. Should we add `pip install -r requirements.txt` to the top of this script? Should this be done in a virtualenv? Should we just update the documentation to say "install requirements with `pip install -r requirements.txt`"?

The requirements installs a surprising number of packages, so I guess I'd prefer that this script were run in a virtual environment, with requirements installed into the venv only. Options include:

- python3 -m venv .env
- [rye](https://github.com/mitsuhiko/rye)
- [pipenv](https://pipenv.pypa.io/en/latest/index.html)

Reply:
We already had instructions int the setup to install the requirements. I added a recommendation to install into a virtual environment with these recommendations but don't think we want to be forceful about installing them (and how and leave that up to the user. 
<img width="969" alt="image" src="https://github.com/TIDES-transit/TIDES/assets/794787/9a5a5163-d81a-4107-9a47-14160c555c8b">